src/scss/components/topnav/index.scss

```diff
@@ -0,0 +1,196 @@
+// ----- BASE ELEMENTS ---------------
+@import "vars";
+@import "./config";
+
+#hxTopnav {
```

Comment:
Please check the redline specs, and make the font consistent:

<img width="843" alt="Screen Shot 2020-10-01 at 5 11 16 PM" src="https://user-images.githubusercontent.com/10120600/94868689-4a23a980-0409-11eb-8cc3-2c553e18fcfe.png">

Reply:
Redline specs are as per the document, and please look at the latest commit for font changes.